packages/edit-site/src/components/style-book/constants.ts

```diff
@@ -216,27 +216,25 @@ export const STYLE_BOOK_IFRAME_STYLES = `
 	}
 	.edit-site-style-book__duotone-example > div:not(:first-child) {
 		height: 20px;
-		border: 1px solid #ddd;
+		border: 1px solid color-mix( in srgb, currentColor 10%, transparent );
```

Comment:
Oooh I didn't know we were already using `color-mix`! That does make things easier.